### PR TITLE
ECF-1-493 Admin school overview page

### PR DESCRIPTION
**Pull request by amazimbe**
### Context
Changes to allow a DfE admin user to see details of a school on a new school overview page. 

### Review Checks
- [x] All pages have automated accessibility checks via cypress
- [x] All `School` queries are correctly scoped - `eligible` when they need to be
image (1)

### How can I view this in a review app?
Go to:  https://ecf-review-pr-262.london.cloudapps.digital/admin

Sign in as admin@example.com

Then from the list of schools click on a name and that should take you to the school overview page for that school.

For a school with an induction coordinator search for "Example school" and click on the name

### CODE REVIEW

**Comment by amazimbe** [comment]
[School detail: Overview](https://trello.com/c/AnO61Iv4/493-school-detail-overview)

**Comment by ethanmills** [comment]
Checking with Al, but I think this page will need a back button

File: app/views/admin/schools/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <td class="govuk-table__cell">Induction tutor</td>
+      <td class="govuk-table__cell">
+        <% if @induction_coordinator %>
+          <%= @induction_coordinator.full_name %><br>
+          <%= mail_to @induction_coordinator.email, @induction_coordinator.email  %>
+        <% end %>
+      </td>
+      <td class="govuk-table__cell">
+        <% if @induction_coordinator %>
+        <%= govuk_link_to "Change", edit_admin_induction_coordinator_path(@induction_coordinator) %>
```
Comment (ethanmills) [inline app/views/admin/schools/show.html.erb:18]: I don't think this should link here. We're not implementing this bit quite yet, but the spec for changing induction tutor is creating a new account (retaining the old one for audit purposes). Let's leave it as a dead link for now?

**Comment by ethanmills** [comment]
Could you add review app steps please? I don't think you'll need to add any seed data, should have everything you need already

**Comment by ethanmills** [comment]
No back link required @amazimbe, sorry, my mistake

**Comment by amazimbe** [comment]
> No back link required @amazimbe, sorry, my mistake

@ethanmills I find that a bit odd. If ever a page needed a back link then this would be it, in my opinion. What was the justification for not having a back link?

**Review by ethanmills — APPROVED** [review]
LGTM

File: spec/cypress/integration/admin/SchoolManagment.feature [hunk 2]
```diff
@@ -6,6 +6,13 @@ Feature: Admin user managing schools
     And scenario "admin/schools" has been run
     And I am on "admin schools" page
 
+  Scenario: Viewing a school
+    When  I click on "link" containing "Include this school"
```
Comment (kicferk1) [inline spec/cypress/integration/admin/SchoolManagment.feature:10]: Is there some double space here? :O

**Review by kicferk1 — APPROVED** [review]
Nitpick about hidden whitespace

**Review by goodviber — CHANGES_REQUESTED** [review]

File: app/views/admin/schools/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          <%= @induction_coordinator.full_name %><br>
+          <%= mail_to @induction_coordinator.email, @induction_coordinator.email  %>
+        <% end %>
+      </td>
+      <td class="govuk-table__cell">
+        <% if @induction_coordinator %>
+        <!-- TODO: Add link to change induction coordinator for a school -->
+        <%= govuk_link_to "Change", "#" %>
+        <% else %>
+        <%= govuk_link_to "Add induction tutor", new_admin_induction_coordinator_path %>
```
[the remark this answers is not in the document]
Comment (amazimbe) [reply app/views/admin/schools/show.html.erb:21]: @goodviber For what I had to do this was good enough. I agree we need the school but to me that's so dependent on how the add induction coordinator has been designed to work. What works for you /admin/induction-coordinator/new?school_id=xxxx or /admin/schools/:id/induction-coordinator/new ....

**Review by goodviber — APPROVED** [review]

**Pull request merged**